shared/syncer.js

```diff
@@ -24,7 +24,8 @@ if (isServer) {
   var serverOnly_qs = 'qs';
   var qs = require(serverOnly_qs);
 } else {
-  Backbone.$ = window.$ || require('jquery');
+  var $ = window.$ || require('jquery');
```

Comment:
I don't think this is necessary, can you explain why you decided to change this?